src/main/java/homeworks/daniel/firstAssigment/interfaceExercise/VideoFile.java

```diff
@@ -0,0 +1,30 @@
+package homeworks.daniel.firstAssigment.interfaceExercise;
+
+public class VideoFile implements MediaSource, DynamicSource {
```

Comment:
If all classes that implement MediaSource also Implement DynamicSource, those two might be just one interface.